dev/ci/README-users.md

```diff
@@ -75,21 +79,21 @@ Have a look at [#7656](https://github.com/coq/coq/pull/7656/files) for an
 example. **Do not hesitate to submit an incomplete pull request if you need
 help to finish it.**
 
-You may also be interested in having your development tested in our
+You may also be interested in having your project tested in our
 performance benchmark. Currently this is done by providing an OPAM package
 in https://github.com/coq/opam-coq-archive and opening an issue at
 https://github.com/coq/coq-bench/issues.
 
 ### Recommended branching policy.
 
 It is sometimes the case that you will need to maintain a branch of
-your development for particular Coq versions. This is in fact very
-likely if your development includes a Coq ML plugin.
+your project for particular Coq versions. This is in fact very likely
+if your project includes a Coq ML plugin.
 
-We thus recommend a branching convention that mirrors Coq's branching
-policy. Then, you would have a `master` branch that follows Coq's
-`master`, a `v8.8` branch that works with Coq's `v8.8` branch and so
-on.
+For such projects, we recommend a branching convention that mirrors
+Coq's branching policy. Then, you would have a `master` branch that
+follows Coq's `master`, a `v8.8` branch that works with Coq's `v8.8`
```

Comment:
I thought this was no longer the recommended policy, and the policy was to instead use `coq-v8.8` or `coq-8.8` or `$project_version-coq-8.8` or `$project_version-coq-v8.8`?

Reply:
I think that this was introduced by @ejgallego and that the exact recommendation was debated at the time and not really settled. I believe that the use of the exact same branch name is still what @ejgallego's tools expect. My preference would indeed be to recommend using the `coq-` prefix and have tools support all these standards. Anyway, that is kind of unrelated to the current PR, which is not modifying the content here.

Reply:
At least for ML plugins, I still prefer identical names; I think @JasonGross 's 4 choices are a good example on why using something different may lead to confusion.